doc/api/next_api_changes/2018-04-20-AL.rst

```diff
@@ -0,0 +1,5 @@
+``bar`` (resp. ``barh``) no longer accepts ``left`` (resp. ``bottom`` as first named argument
```

Comment:
`resp`??? and missing closing bracket after "bottom"...

Reply:
See entry for "resp." at https://en.wikipedia.org/wiki/List_of_mathematical_jargon :-)
Anyways, switched to a simpler wording.

Reply:
I am illiterate!  And I even did a math degree (at some point, long ago, not very well...)